feat/new product overview pages (#646)
hirosystems/docs: docs/platform/index.mdx

---
# The default id is the same as the one defined below. so not needed
title: Hiro Platform Overview
sidebar_label: Overview
custom_edit_url: null
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

<p class="p-3 rounded-lg bg-neutral-100 text-neutral-500 dark:bg-neutral-800 dark:text-neutral-400">A cloud-based development environment to kickstart your app development on Stacks.</p>

<div class="mb-10"></div>

The Hiro Platform is a hosted developer experience for creating, testing, and deploying smart contracts. It comes pre-installed with several tools, including Clarinet, VS Code, and a private blockchain network for testing, so you can spend less time setting up and more time building.

> You can also set up custom event streams via chainhooks and manage API keys for Stacks and Ordinals.

Reviewer: ```suggestion
You can also set up custom event streams via chainhooks and manage API keys for Stacks and Ordinals.

Spin up your next project in just a few clicks.
```

    <p>Create and manage webhook-like triggers that react to on-chain events in real time.</p>
  </a>
  <a href="/platform/guides/deployment-plans">
    <div class="subSectionTitle"><h3>Deploy your code</h3><span>→</span></div>
    <p>Deploy your project with customizable deployment plans that can handle contract dependencies.</p>
  </a>
</div>

<div class="subSections my-8 w-[50%]">
  <a href="/platform/guides/devnet">
    <div class="subSectionTitle"><h3>Iterate rapidly with Devnet</h3><span>→</span></div>
    <p>Leverage devnet, a private instance of the blockchain, for rapid feedback loops and a better debugging experience.</p>
  </a>
</div>

### Related Tools

**[Clarinet](../clarinet/index.mdx)**: Prefer to stay local? Build and deploy smart contracts on your device with Clarinet. 

**[Chainhook](../chainhook/index.mdx)**: You don’t need the platform to create event streams. Leverage chainhook independently, so your app can react to on-chain events in real time. 

**[Stacks.js](../stacks.js/index.mdx)**: Don’t reinvent the wheel and build faster with a JS library that handles basic functions.

---

> Need help building with Clarinet? Reach out to us on the `#hiro-platform` channel on Discord under Hiro Developer Tools section. We also host a weekly office hours on Discord every Monday at 11am ET.

Reviewer: ```suggestion
Need help building with Clarinet? Reach out to us on the `#hiro-platform` channel on Discord under the [Hiro Developer Tools section](https://stacks.chat/). There’s also a [weekly office hours](https://events.stacks.co/event/HD16484710) on Discord every Thursday at 11am ET.
```
